cookbook/live-preview.rst

```diff
@@ -0,0 +1,171 @@
+                </properties>
+            </section>
+        </properties>
+    </template>
+
+This page consists of a single title and a url. The template is quite basic but
+includes the RDFa attribute `property=title` to tell sulu that the title is
+rendered in this container.
+
+When the title of the page will be updated in the content form the admin will
+send this change to the server where the twig block `content` will be rendered.
+The server response contains only the part which was changed (indicated by
+`property=title`). Back in the javascript application the preview on the right
+will update the container which is marked with the particular property.
+
+This process ensures a quite good performance (as good as the rendering on the
+server side) and minimizes the load of the requests.
+
+Supported RDFa features
+-----------------------
+
+To see a full example of the syntax take a look in the
+`example.html.twig <https://github.com/sulu/sulu-standard/blob/develop/src/Client/Bundle/WebsiteBundle/Resources/themes/default/templates/example.html.twig>`
+file.
+
+Links
+*****
+
+For links you can use the href-attribute which will also be updated if the
+mentioned property was changed.
+
+.. code-block:: twig
+
+    <a href="{{ sulu_content_path(content.link.url) }}" property="link">
+        {{ content.link.title }}
+    </a>
+
+Images
+******
+
+To update an image you can simply use the src-attribute when the mentioned
+property has changed
+
+.. code-block:: twig
+
+    <img src="{{ image.thumbnails['170x170'] }}" alt="{{ image.title }}"/>
+
+
+Multiple values
+***************
+
+For multiple values simply use table, ul, ol or div tags and the content will
+be updated if the mentioned property was changed.
+
+.. code-block:: twig
+
+    <ul property="categories">
+        {% for category in content.categories %}
+            <li>{{ category.name }}</li>
+        {% endfor %}
+    </ul>
+
+Snippets
+********
+
+For the snippet content-type all the selected snippets has to be updated.
+Therefore you only have to set the upper property.
+
+.. code-block:: twig
+
+    <div property="snippets">
+        {% for snippet in content.snippets %}
+            <h2>{{ snippet.title }}</h2>
+        {% endfor %}
+    </div>
+
+Blocks
+******
+
+Blocks are the only property-type which needs a different syntax beside the
+`property` attribute.
+
+.. code-block:: twig
+
+    <div property="blocks" typeof="collection">
+        {% for block in content.blocks %}
+            <div rel="blocks" typeof="block">
+                <div property="title">{{ block.title }}</div>
+            </div>
+        {% endfor %}
+    </div>
+
+You have to define the `property="blocks"` as `typeof="collection"` and each
+item of the block as `typeof="block"` and set the relation to the parent
+property in this case "blocks".
+
+With this definitions the system is able to update only the `title` of the
+first block item and don't response the whole div-container of the
```

Comment:
`and doesn't have to return the entire container of the block-property in the response`